### PR TITLE
join testnets - remove duplicated content

### PR DESCRIPTION
**Pull request by andreipope**
We ended up having somehow duplicated content in "Intro to Loom" and "Join Testnets".

This moves the new content to the "Join Testnets" page and removes the old content.

### CODE REVIEW

File: docs/en/intro-to-loom.md [hunk 1]
```diff
@@ -21,166 +21,3 @@ So, the Loom SDK allows for developers to spin up an unlimited number of sidecha
 But PlasmaChain will be the central hub that all these chains connect to as their bridge to Ethereum, for interchain communication with other major blockchains (such as Cosmos, EOS, and Tron), and as a shared liquidity pool for tokenized assets.
 
 Thus, our company’s primary objectives over the next year are all related to growing PlasmaChain to become the “universal Layer 2” for high performance dapps.
-
-In the next chapter, you will learn how to deploy your first app to Loom PlasmaChain. Let's get started!
```
Comment (mattkanwisher) [inline docs/en/intro-to-loom.md]: Shouldn’t it still lead into the next article ?

File: docs/en/join-testnet.md [hunk 2]
```diff
@@ -4,58 +4,168 @@ title: Join Testnets
...
+yarn gen:loom-key
+```
+
+### Spin up Loom Chain
+
+In a new terminal, run:
 
 ```bash
-./loom genkey -k priv_key -a pub_key
+yarn loom-init
```
Comment (mattkanwisher) [inline docs/en/join-testnet.md]: Does this new command require loom?

File: docs/en/join-testnet.md [hunk 3]
```diff
@@ -4,58 +4,168 @@ title: Join Testnets
...
+yarn test
 ```
 
-This will create two files named `priv_key` and `pub_key`. The `priv_key` file contains your private key that you'll use to deploy contracts to the DAppChain.
+### Web interface
 
+Open the `src/contract.js` file in your text editor of choice and go to line 87.
 
-### Deploy Truffle Example to a Testnet
+Now, say your `Network id` is `13654820909954`. Replace
```
Comment (mattkanwisher) [inline docs/en/join-testnet.md]: Why is this needed ?

Comment (andreipope) [reply docs/en/join-testnet.md]: It tries to connect with `default` but this doesn't work. It just errors out so I added a comment to the source code and this paragraph to the docs. Can't remember if I tried with `*` instead of `default`. I'll give it a shot.

https://github.com/loomnetwork/truffle-dappchain-example/blob/0ee45592908e32468f96d0f7a406b190dddb98a0/src/contract.js#L87

File: docs/en/join-testnet.md (hunk 2 above)
Comment (andreipope) [reply docs/en/join-testnet.md]: > Does this new command require loom?

Yes, it initializes loom and then spins up the blockchain: "loom:init": "loom/loom init -f && loom/loom run".

Anyway, it's `loom:init`. I'll fix the typo.

File: docs/en/intro-to-loom.md (hunk 1 above)
Comment (andreipope) [reply docs/en/intro-to-loom.md]: > Shouldn’t it still lead into the next article ?

I wasn't sure if it's okay to refer to the "Join Testnets" page as the next chapter.  But probably it's just fine and it makes a nice transition. I'll bring it back.

File: docs/en/join-testnet.md (hunk 3 above)
Comment (andreipope) [reply docs/en/join-testnet.md]: Tried with `*`... it doesn't work either. Maybe I should dig into this a bit more and figure out a better way of doing it.
